src/MCPClient/lib/clientScripts/storeAIP.py

```diff
@@ -57,6 +58,21 @@ def get_upload_dip_path(aip_path):
     return os.path.sep + os.path.join(*new_aip_path)
 
 
+def rmtree_upload_dip_transitory_loc(package_type, unit_path):
+    """If a DIP has been stored but still exists in the DIP Upload watched
+    directory then it needs to be deleted.
+    """
+    if package_type == "DIP":
+        unit_path = get_upload_dip_path(unit_path)
+        LOGGER.info(
+            "'%s' Stored removing duplicates in watched dir: %s",
+            package_type, unit_path)
```

Comment:
So this would print something like:

    'DIP' Stored removing duplicates in watched dir: /dadada...

I'm not a native speaker so feel free to ignore, but is this grammatically correct? Did you forget a comma?
Also, when sentence case is used only the first word is capitalized right? One more thing, as you already know for sure that `package_type="DIP"`, do you still need to expand the variable?